### DISCUSSION TITLE
Labels' alignment in checkboxes and radio-buttons

### DISCUSSION THREAD

**Discussion by turolopezsanabria** [SUI-Components/sui-components#1566] [Ideas and new features]
**Absolutely related with this other discussion:**
https://github.com/SUI-Components/sui-components/discussions/1558

Ok, `labels` in checkboxes and radio-buttons have a pretty crazy alignments right now, specially when multiline

**What do we do with them?**

Can we go with a single solution? Should it be customisable via Prop?

1. Top
2. Middle

![labels](https://user-images.githubusercontent.com/23620759/121935564-da600f80-cd48-11eb-8b57-17961ba13a5b.png)

**Comment by ChrisDesignOps** [comment]
I think that the Label always have to be aligned to the first line centered with the check/radio to ease the readability.
If you center the content as in the 1st example and the label had more than 3 lines or maybe 4 in mobile, the action would be visually lost (well, in the case of RS with those huge checks not xD)

**Comment by turolopezsanabria** [comment]
fixed
